guides/common/modules/proc_importing-templates-using-ansible.adoc

```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="importing-templates-using-ansible"]
+= Importing templates using Ansible
+
+You can import templates from a repository of your choice.
+You can use different protocols to point to your repository, for example `/tmp/dir`, `git://example.com`, `\https://example.com`, and `ssh://example.com`.
+
+.Procedure
+* Use the `{ansible-namespace}.templates_import` module.
+
+.Additional resources
+* For more information, see the Ansible module documentation with `ansible-doc {ansible-namespace}.templates_import`.
```

Comment:
This would not work with DITA. Links in additional resources must be actual links, without any text around them. So perhaps:

```suggestion
* [https://galaxy.ansible.com/ui/repo/published/evgeni/foreman/content/module/templates_import/](templates_import in Ansible Galaxy documentation).
```